games/devtest/mods/testnodes/properties.lua

```diff
@@ -209,6 +209,41 @@ minetest.register_node("testnodes:climbable", {
 	groups = {dig_immediate=3},
 })
 
+minetest.register_node("testnodes:climbable_fast", {
+	description = S("Fast Climbable Node").."\n"..
+		S("You can climb up and down, faster than usual"),
+	climbable = true,
+	climb_factor = 2.0,
+	walkable = false,
+
+
+	paramtype = "light",
+	sunlight_propagates = true,
+	is_ground_content = false,
+	tiles = {"testnodes_climbable_top.png^[colorize:#FFFFFF:140","testnodes_climbable_top.png^[colorize:#FFFFFF:140","testnodes_climbable_side.png^[colorize:#FFFFFF:140"},
```

Comment:
```suggestion
	tiles = {
		"testnodes_climbable_top.png^[colorize:#FFFFFF:140",
		"testnodes_climbable_top.png^[colorize:#FFFFFF:140",
		"testnodes_climbable_side.png^[colorize:#FFFFFF:140"
	},
```

We want lines to be wrapped at 80 characters.